.drone.star

```diff
@@ -94,6 +94,7 @@ config = {
                 "apiDepthInfinity",
                 "apiLocks",
                 "apiActivities",
+                "apiOptionalShareRole",
```

Comment:
this should go to the sharingNG suite. we can think of spliting the bulky ones later